### PR TITLE
ipn/ipnstate: put SSH Host Keys in ipnstate.PeerStatus

### PR DESCRIPTION
**Pull request by bradfitz**
Updates #3802

### CODE REVIEW

File: ipn/ipnstate/ipnstate.go [hunk 1]
```diff
@@ -145,6 +145,9 @@ type PeerStatus struct {
 	PeerAPIURL   []string
 	Capabilities []string `json:",omitempty"`
 
+	// SSH_HostKeys are the node's SSH host keys, if known.
+	SSH_HostKeys []string `json:"sshHostKeys,omitempty"`
```
Comment (maisem) [inline ipn/ipnstate/ipnstate.go:149]: views.Slice[string]?

Comment (bradfitz) [reply ipn/ipnstate/ipnstate.go:149]: I'd rather do them all at the same time. For now I value consistency.

Comment (maisem) [reply ipn/ipnstate/ipnstate.go:149]: we kinda already are inconsistent here, but sure.

Comment (bradfitz) [reply ipn/ipnstate/ipnstate.go:149]: Also, you'd want the pointer. Above we do have one `Tags *views.Slice[string]`

Comment (maisem) [reply ipn/ipnstate/ipnstate.go:149]: e.g. Tags and PrimaryRoutes

**Review by maisem — APPROVED** [review]

File: ipn/ipnstate/ipnstate.go (hunk 1 above)
Comment (bradfitz) [reply ipn/ipnstate/ipnstate.go:149]: I really wish we didn't need that pointer. Ugh.

/cc @dsnet (for omitzero in json)

Comment (maisem) [reply ipn/ipnstate/ipnstate.go:149]: yeah you'd need the pointer for `omitempty` to work

**Pull request merged**